src/Harpoon.cpp

```diff
@@ -13,7 +13,7 @@
 
 /// Creates the client that is to connect to the server/Kraken
 Harpoon::Harpoon(): 
-mQueueLength(10),
+mQueueLength(1),
```

Comment:
What does that mean?  What is 10, what is 1?  Just the number of input messages?